.readthedocs.yaml

```
@@ -5,25 +5,30 @@
 # Required
 version: 2
 
-build:
-  os: ubuntu-24.04
-  tools:
-    python: "3.10"
-  jobs:
-    post_checkout:
-      - git fetch --tags --depth 1  # Also fetch tags
-      - git describe --always       # Useful for debugging
-
 # Build documentation in the docs/ directory with Sphinx
 sphinx:
   configuration: docs/conf.py
   builder: dirhtml
   fail_on_warning: false
 
-# Optionally set the version of Python
-# and requirements required to build your docs
-python:
-  install:
-    - requirements: requirements-dev.txt
-    - method: pip
-      path: .[docs]
+# Optionally build your docs in additional formats such as PDF
+# formats:
+#  - pdf
+#  - epub
+
+build:
+  os: ubuntu-24.04
+  tools:
+    python: "3.12"
+  jobs:
+    post_checkout:
+      - git fetch --tags --unshallow # Also fetch tags
+      - git describe --always       # Useful for debugging
```

```suggestion
      - git fetch --tags --unshallow  # Also fetch tags
      - git describe --always         # Useful for debugging
```
This (should?) fix a lint warning about "too few spaces before comment"